test/src/main/java/org/apache/accumulo/test/functional/ManyWriteAheadLogsIT.java

```diff
@@ -71,6 +77,38 @@ protected int defaultTimeoutSeconds() {
     return 10 * 60;
   }
 
+  @Before
+  public void alterConfig() throws Exception {
+    if (getClusterType() == ClusterType.MINI) {
+      return;
+    }
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      InstanceOperations iops = client.instanceOperations();
+      Map<String,String> conf = iops.getSystemConfiguration();
+      majcDelay = conf.get(Property.TSERV_MAJC_DELAY.getKey());
+      walogSize = conf.get(Property.TSERV_WALOG_MAX_SIZE.getKey());
+
+      iops.setProperty(Property.TSERV_MAJC_DELAY.getKey(), "1");
+      iops.setProperty(Property.TSERV_WALOG_MAX_SIZE.getKey(), "1M");
+
+      getClusterControl().stopAllServers(ServerType.TABLET_SERVER);
+      getClusterControl().startAllServers(ServerType.TABLET_SERVER);
+    }
+  }
+
+  @After
+  public void resetConfig() throws Exception {
+    if (majcDelay != null) {
+      try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+        InstanceOperations iops = client.instanceOperations();
+        iops.setProperty(Property.TSERV_MAJC_DELAY.getKey(), majcDelay);
+        iops.setProperty(Property.TSERV_WALOG_MAX_SIZE.getKey(), walogSize);
+      }
+      getClusterControl().stopAllServers(ServerType.TABLET_SERVER);
+      getClusterControl().startAllServers(ServerType.TABLET_SERVER);
+    }
+  }
+
```

Comment:
This is interesting.  I don't think we have anything like this in our ITs yet.  I can see why you would need to reset the config on the standalone instance.  I am just now wondering if there is a better way to do this for all ITs...

Reply:
@ctubbsii  and @milleruntime Wish I could take credit but I did see this elsewhere (e.g., BinaryStressIT).  Just let me know what the decision is whether to keep these as standalone tests or not.

Reply:
I think what you did is OK for now.  We may have to go back later and reevaluate which tests we want to be Standalone or Mini.